tests/Application/config/routes/sylius_shop.yaml

```diff
@@ -2,7 +2,7 @@ sylius_shop:
     resource: "@SyliusShopBundle/Resources/config/routing.yml"
     prefix: /{_locale}
     requirements:
-        _locale: ^[A-Za-z]{2,4}(_([A-Za-z]{4}|[0-9]{3}))?(_([A-Za-z]{2}|[0-9]{3}))?$
+        _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
```

Comment:
Is this change needed for Sylius 1.12?

Reply:
It was not needed, restored to old value in next commit